docs/installation/presto-admin-installation.rst

```diff
@@ -8,6 +8,13 @@ Installing Presto Admin
 .. NOTE::
      For Amazon EMR, use the online installer instead of the offline installer (see :ref:`advanced-installation-options-label`).
 
+.. NOTE::
+   Omit the use of ‘sudo’ when the prestoadmin scripts and presto-admin commands when the root user is selected as the prestoadmin user.
+
+.. NOTE::
+   The sudo setup for a non-root user as the prestoadmin has a security hole where the prestoadmin non-root user has the ability to run bash as root.  The IT organization will need to take the appropriate steps to address this security hole and select which user will be the prestoadmin user..
```

Comment:
omit "as the prestoadmin"
Suggest:
The sudo setup for a non-root user has a security hole where the presto-admin non-root user requires the ability to run /bin/bash as root.  The IT organization will need to take the appropriate steps to address this security hole and select which user will be the presto-admin user.